e2e/atlas/clusters_test.go

```diff
@@ -121,6 +121,24 @@ func TestClustersFlags(t *testing.T) {
 		}
 	})
 
+	t.Run("Create Rolling Index", func(t *testing.T) {
```

Comment:
moving this before update, we get random 500 from it and I guess is because the cluster is still updating